src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/HistoricalBeneficialOwnerValidator.java

```diff
@@ -27,8 +27,11 @@ public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, Strin
             List<HistoricalBeneficialOwnerDto> historicalBeneficialOwners = trustDataDto
                     .getHistoricalBeneficialOwners();
             if (!CollectionUtils.isEmpty(historicalBeneficialOwners)) {
+                boolean isUnableToObtainAllTrustInfo = trustDataDto.getUnableToObtainAllTrustInfo();
                 for (HistoricalBeneficialOwnerDto historicalBeneficialOwnerDto : historicalBeneficialOwners) {
-                    validateCeasedDate(historicalBeneficialOwnerDto.getCeasedDate(), errors, loggingContext);
+                    if (!isUnableToObtainAllTrustInfo) {
```

Comment:
Noticed in the chips database trust table that it is possible to have unableToObtainAllTrustinfo as N and have a null cease date would a further condition be required here?

Reply:
Are you referring to the trust ceased date? In which case it's perfectly valid to not have a ceased date. This validation refers to the trustee ceased date for a trustee with type 5001 (historical BO).